dashboard/test/models/pd/workshop_test.rb

```diff
@@ -189,37 +189,45 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
   end
 
   # Tests the following:
-  #   1. enrolled_not_attended teacher does not get an email
-  #   2. enrolled_attendee and unenrolled_attendee each get an email
+  #   1. enrolled-only teacher does not get an email
+  #   2. All teachers in the section get an email
   #   3. accidental_student_attendee is converted to a teacher account and gets an email
```

Comment:
Good suggestion. I probably did it this way initially just to make sure it works with a set of enrollments not just individuals. Separate tests would be easier to read and to debug if they ever fail. I'll split it up.